NEWS

```diff
@@ -2,6 +2,13 @@ systemd System and Service Manager
 
 CHANGES WITH 238:
 
+        * NETWORK INTERFACE DEVICE NAMING CHANGES: systemd-udevd's "net_id"
+          builtin may name network interfaces diferently than in previous
+          versions.  SR-IOV virtual functions and NPAR partitions with PCI
+          function numbers of 8 and above will be named more predictably,
+          and udev may generate names based on PCI slot number in some cases
+          where it previously did not.
```

Comment:
238 got release in the meantime. Please rebase on current git, and add this to a new section for "239 in spe"